doc/management/private.md

```diff
@@ -1,9 +1,178 @@
 # Private API for Management
 
+## Enum `Status`
+
+The status enumeration contains the following possible values:
+
+* `NOT_REGISTERED`
+* `REGISTERED`
+* `REGISTERED_GOT_MATERIAL`
+* `DEREGISTERED`
+
+## Data type `Event`
+
+Properties:
+
+* `id`: The ID of the event
+* (optional) `identifier`: The identifier of the event
```

Comment:
id and identifier are synonyms. I suggest "id" and "referenceId"

Reply:
I accociate the term `referenceId` with something else. Maybe just indicate that it is not an ID at all. Maybe `alias` is a better word.

Reply:
"referenceId" is usually the ID of a different system, which is exactly the case here. "externalId" would be also possible, but an alias is something different.